### DISCUSSION TITLE
no parameter to specify compartibility for java versions

### DISCUSSION THREAD

**Discussion by P1tt187** [renovatebot/renovate#25411] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.32.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I use Renovate to provide an internal docker container in my company.
I use the `java-version` datasource to determine the version.
Currently there is no option to specify comparitbilty parameters.

In my case i need binarys for linux x64.
This leads to renovate offering the update too early.

A good example is the following release page:
https://github.com/adoptium/temurin8-binaries/releases/tag/jdk8u392-b08

The first artefacts where uploaded on 18 October, the artifacts I need where available on 23 October.
So I got a pullrequest that cannot build for 5 days.

It would be nice append the parameters `architecture` and `os`. The Parameters are part of the adoptium api.
In this case renovate would offer the update only when it is available for the platform i use.

### What you tried so far.

I Specified the version in a file and use a regex manager to find new versions, example here:

```
# renovate: datasource=java-version depName=java-jdk
JDK_8_VERSION="8.0.382+5"
```

**Comment by rarkins** [comment]
@viceice do we do anything similar for Containerbase or is the only solution to add a minimumReleaseAge with strict filtering?

**Reply by viceice** [reply]
I think we didn't check yet. we probably should extend the java-version datasource together with the other refactor we've planned.
